airflow/www/views.py

```diff
@@ -1325,7 +1325,7 @@ def xcom(self, session=None):
         dm_db = models.DagModel
         ti_db = models.TaskInstance
         dag = session.query(dm_db).filter(dm_db.dag_id == dag_id).first()
-        ti = session.query(ti_db).filter(ti_db.dag_id == dag_id and ti_db.task_id == task_id).first()
+        ti = session.query(ti_db).filter(and_(ti_db.dag_id == dag_id, ti_db.task_id == task_id)).first()
```

Comment:
For future reference, this would also work:

```python
         ti = session.query(ti_db).filter(ti_db.dag_id == dag_id, ti_db.task_id == task_id).first()
```

Reply:
Good to know! Thanks